src/server/auth.ts

```diff
@@ -74,3 +96,13 @@ export const authOptions: NextAuthOptions = {
  * @see https://next-auth.js.org/configuration/nextjs
  */
 export const getServerAuthSession = () => getServerSession(authOptions);
+
+export const getRequiredServerAuthSession = async () => {
```

Comment:
utility to narrow session type

Reply:
Since this is not being used anywhere, i assume you created this so we dont have to keep on checking and trowing exception in places? If that is the case, perhaps naming it something like `withServerSession` (similar to `getServerSession`

If it has a different use case, please let me know.